Packages/com.unity.inputsystem/InputSystem/Controls/KeyControl.cs

```diff
@@ -50,7 +51,13 @@ protected override void RefreshConfiguration()
             if (device.ExecuteCommand(ref command) > 0)
             {
                 m_ScanCode = command.scanOrKeyCode;
-                displayName = command.ReadKeyName();
+
+                var rawKeyName = command.ReadKeyName();
+                var textInfo = CultureInfo.CurrentCulture.TextInfo;
```

Comment:
Should we use InvariantCulture instead? From the docs, it doesn't really make a difference now but it might in the future. The docs also have one of my favourite lines of documentation: "We reserve the right to make this API slower in the future."

Reply:
Fixed